docs/src/pages/guides/classname-generator/classname-generator.md

```diff
@@ -105,3 +106,66 @@ Now, the `Mui` class is gone.
 
 - This API should only be used at build-time.
 - The configuration is applied to all of the components across the application. You cannot target a specific part of the application.
+
+## Framework Example
+
+In some cases, you might need to add `/* eslint-disable import/first */` at the top of `ClassNameGenerator` import.
+
+```js
+/* eslint-disable import/first */
+import { unstable_ClassNameGenerator as ClassNameGenerator } from '@mui/material/utils';
+```
+
+### Create react app
+
+use ClassNameGenerator in `/src/index.js`.
+
+```diff
++import { unstable_ClassNameGenerator as ClassNameGenerator } from '@mui/material/utils';
+
++ClassNameGenerator.configure((componentName) => componentName.replace('Mui', ''));
+
+import * as React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+ReactDOM.render(
+  <App />,
+);
+```
+
+### NextJS
```

Comment:
```suggestion
### Next.js
```

Reply:
I would personally put this one before CRA, I believe it's more popular nowadays (if you look at the npm download growth trend) (it's put first in https://mui.com/getting-started/example-projects/#official-examples :p)